backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import numpy as np
+
+from nodes.properties.inputs import ImageInput, SliderInput
+from nodes.properties.outputs import ImageOutput
+
+from .. import adjustments_group
+
+
+@adjustments_group.register(
+    schema_id="chainner:image:clamp",
+    name="Clamp",
+    description="Clamps the values of an image.",
+    icon="ImContrast",
+    inputs=[
+        ImageInput(),
+        SliderInput(
+            "Minimum",
+            minimum=0.0,
+            maximum=1.0,
```

Comment:
Same old: 0-1, 0-100, or 0-255? What do you think would be best here?

Reply:
I think we should slowly consider a move to a normalized / float (0.0 - 1.0 in this case) Ui paradigm for denoting image bit-depth and applicable and relevant operational value ranges.

Reply:
Man, we really need a way for users to select the unit they want, and we always get 0-1. 

Let's keep it as is for now.